src/wwwroot/js/genpage/helpers/browsers.js

```diff
@@ -107,6 +107,39 @@ class GenPageBrowserClass {
         this.isSmallWindow = mobileDesktopLayout == 'auto' ? window.innerWidth < 768 : mobileDesktopLayout == 'mobile';
     }
 
+    getFilterTooltip() {
+        if (this.id !== 'imagehistorybrowser') {
+            return 'Text filter, only show items that contain this text.';
+        }
+        return `SEARCH FILTERS:
+
+Text: any text (searches everywhere)
+prompt:woman (in prompt)
+lora:name (lora containing name)
+model:sdxl (model containing sdxl)
+controlnet:canny (controlnet type)
+sampler:euler (sampler name)
+aspectratio:16:9 (aspect ratio)
+
+NUMERIC FILTERS (support >, <, =):
```

Comment:
symbols like this fed directly into an html attrib unescaped could cause problems